step-controller/step-controller-base-plugins/src/main/java/step/plugins/parametermanager/ParameterManagerControllerPlugin.java

```diff
@@ -63,7 +70,11 @@ public void serverStart(GlobalContext context) {
 		context.put("ParameterAccessor", parameterAccessor);
 
 		context.get(TableRegistry.class).register(ENTITY_PARAMETERS, new Table<>(collection, "param-read", true)
-				.withResultItemEnricher(p -> ParameterServices.maskProtectedValue(p)));
+				.withResultItemEnricher(ParameterServices::maskProtectedValue)

```

Comment:
Should this really be an enricher? Enrichers can be disabled by the user. To unconditionally apply the masking, you can use the transformer instead of enricher.

Reply:
@cl-exense well this is pretty bad, because this is the legacy implementation of result enricher and the parameter table. I imagine making the enricher optional is recent and the parameter table hasn't been updated accordingly.

Reply:
No, the enricher has always been "optional" in the sense that it can be disabled, as far as I know. That was the whole reason for introducing the transformer function which cannot be disabled.